hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueFactory.kt

```diff
@@ -103,8 +103,20 @@ class HmppsQueueFactory(
     return when (provider) {
       Provider.AWS -> sqsClientFactory.awsSqsAsyncClient(queueConfig.dlqAccessKeyId, queueConfig.dlqSecretAccessKey, region, hmppsSqsProperties.useWebToken, queueConfig.propagateTracing)
       Provider.LOCALSTACK -> {
+        val attributes = when {
+          queueConfig.fifoQueue == "true" -> mapOf(QueueAttributeName.FIFO_QUEUE to queueConfig.fifoQueue, QueueAttributeName.FIFO_THROUGHPUT_LIMIT to queueConfig.fifoThroughputLimit) else -> mapOf()
+        }
```

Comment:
It would be a good idea to verify that the supplied queue names end in `.fifo` at this point

Reply:
What's the significance of a `.fifo` suffix?

If you do want that verification then you should probably add some validation for it to HmppsSqsProperties.

Reply:
FIFO queues and topics require the `.fifo` suffix